.github/workflows/test.yml

```diff
@@ -19,4 +26,4 @@ jobs:
     - name: Install dependencies
       run: bundle install
     - name: Run test
```

Comment:
Opportunity: can use setup-ruby's `bunder-cache: true` and omit the `bundle install` step.

Reply:
👍